components/faqs/ContributeFAQ.js

```diff
@@ -45,6 +45,22 @@ const ContributeFAQ = ({ collective, ...props }) => (
         </Content>
       </Entry>
     )}
+    {collective.type === 'COLLECTIVE' &&
```

Comment:
Testing `collective.type === 'COLLECTIVE'` seems too restrictive. Other valid values could be `FUND`, `EVENT`, `PROJECT` and maybe more in the future. I see no reason to have it.